### DISCUSSION TITLE
Transferred issues not available to add to projects

### DISCUSSION THREAD

**Discussion by attieretief** [community/community#6065] [Projects and Issues]
After transferring an issue from one repo to another in an organisation, it is not possible to add the new issue by number to the new issues/projects feature - it does not appear in the selection list

**Comment by PhoebeHui** [comment]
We encountered same issue, and the transderred issue was almost missed.

It looks the issue ordered by original 'create date' in issue tab. After it transferred to new repo, even the issue number is new, the 'create date' may be older, so we couldn't see it in first page by default, see following picture:

![image](https://github.com/community/community/assets/20694052/09bd32b6-a80d-4778-a047-b560b8104b09)

**Comment by dmitrykozlov** [comment]
Would be nice to add sorting by issue number
